doc/user/content/demos/business-intelligence.md

```diff
@@ -83,12 +83,12 @@ generator's continued activity.
 
 With that data available, you can perform complex queries, and Materialize
 maintains their results, even as the underlying data changes. This means that
-you can get answers to your queries in real time, rather than relying on day-old
+you can get answers to your queries in real-time, rather than relying on day-old
```

Comment:
I think this one is properly unhyphenated, unlike the others, because it’s not being used as an adjective.